idx/bigtable/bigtable.go

```diff
@@ -394,6 +395,12 @@ func (b *BigtableIdx) processWriteQueue() {
 		pre := time.Now()
 		complete := false
 		attempts := 0
+		boff := &backoff.Backoff{
```

Comment:
Also, we can remove `attempts` now, as it is not used anywhere else.

Reply:
or, do as store/bigtable does and regularly log a warning:
```
				if (attempts % 20) == 0 {
					log.Warnf("btStore: Failed to write %d chunks to bigtable. they will be retried. %s", len(rowKeys), err)
				}
```

note: we don't need our own var. can use `boff.Attempt()`